### ISSUE TITLE
Spatial Data on the Web Working Group

### ISSUE THREAD

**Issue by tguild** [w3c/strategy#252]
New charter proposal, reviewers please take note.

# Charter Review

[Charter:]https://w3c.github.io/sdw/roadmap/charter-2020.html

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* New
 - [ ] New WG 
 - [ ] New IG
 - If this is a new WG or IG charter request, link to [Advance Notice](), and any issue discussion: 

* Existing 
 - [ ] Existing WG recharter 
 - [ x] Existing IG recharter
 - If this is a charter extension or revision, link a [diff from previous charter](https://services.w3.org/htmldiff), and any issue discussion:

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security.  Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).

Communities suggested for outreach: 

Known or potential areas of concern: 

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)

This strategy funnel issue

Anything else we should think about as we review? 

Group was a WG, encouraged to become an IG and now again steered toward being a WG. It is a joint group with OGC

**Comment by samuelweiler** [comment]
This appears to have started from other-than-the-most-current [charter template](https://github.com/w3c/charter-drafts/blob/gh-pages/charter-template.html).  There are only a few changes to adopt, but I encourage you to adopt them.

**Comment by tguild** [comment]
@samuelweiler yes, this started as rechartering as an IG, was drawn out with custom joint licensing/copyright, unanswered questions to Strategy and W3M on whether scope could be flexible to reflect potential work, had previous horizontal review and sent to W3M who returned it to recharter as a WG again.

As nothing pleases me more than editing charters in a loop, have updated HR link and s/a section/separate sections/ since it seems already draft had new process doc date (commit before those). I await eagerly the pending template pull requests.

We have not discussed but will likely keep the IG page and update it since there are extensive links.

**Comment by michael-n-cooper** [comment]
Liaison to WAI should be updated to reference [Accessible Platform Architectures WG](https://www.w3.org/WAI/APA/charter). APA is looking into a liaison person. It has an interest in tagging locations for accessibility features / accessible routes, is unclear whether this is in the scope of this charter or somewhere else.

**Comment by akuckartz** [comment]
Maybe GeoSPARQL should be explicitely mentioned in the charter?
https://www.ogc.org/projects/groups/geosparqlswg
https://github.com/opengeospatial/ogc-geosparql

**Comment by samuelweiler** [comment]
@tguild 

> As nothing pleases me more than editing charters in a loop, have updated HR link and s/a section/separate sections/ since it seems already draft had new process doc date (commit before those). I await eagerly the pending template pull requests.

I hear you on the pain.  We talked about this the other day on the strategy call, and I'm sure we'd all welcome ideas that make it easier to create charters while keeping up with the sundry changes from the world.
 
Thank you for manually backporting the recent template changes.  Other reviewers: it looks like this was still based primarily on the old charter, so check your own sections carefully.

@tguild, I added a note above which you might not have seen yet:

> I would appreciate more plain-language clarity about the relationship with JWOC. The opening sentence suggests that the JWOC is an agreement; perhaps link to the JWOC? Or explain the JWOC's role in setting (limiting?) the WG's scope?

Charter history section still has todo tags; at the least, it needs to enumerate the deliverable changes.

Nothing else of note.

**Comment by tguild** [comment]
@samuelweiler 

Server side includes, php, CMS or other ways to keep boilerplate consistent over time and take a static snapshot when group is approved would be one way to handle it. That could also help with duplicate data points such as start/end dates at top and bottom. Updated history as that was missing extension.
 
The JWOC name from chair recollection has been the first name for this group that came out of initial OGC/W3C. As we do not want to upend any agreements we left it alone. When it came up with the chairs, it did prompt me to attempt reciting, not committed to memory.
 
https://www.poetryfoundation.org/poems/42916/jabberwocky

**Comment by tguild** [comment]
Hi @akuckartz while GeoSPARQL certainly relies on W3C SPARQL, it has not been a joint OGC/W3C spec to date.

**Comment by tguild** [comment]
@michael-n-cooper updated to APA. We are coordinating with/monitoring progress of Linked Data for Accessibility CG on location accessibility data. We are not working directly on any routing within SDW although OGC does have a routing pilot project and I have been doing some coordination with them, ISO SmartCities and ISO Intelligent Transportation Systems on ontologies with routing at the top of the list. We can bridge some of that coordination with APA through SDW

**Comment by samuelweiler** [comment]
> @samuelweiler
> ...
> The JWOC name from chair recollection has been the first name for this group that came out of initial OGC/W3C. As we do not want to upend any agreements we left it alone. When it came up with the chairs, it did prompt me to attempt reciting, not committed to memory.

So if JWOC has been subsumed by this group or otherwise no longer exists, this scope sentence is downright misleading:

> Both Spatial Data on the Web WG and JWOC **exist** ....

(emphasis added)

This should be cleared up.

**Comment by tguild** [comment]
@samuelweiler I found some earlier, clearer wording on JWOC. Rephrasing (not mine) across charters has muddled it somewhat. It reads better now.

[[
The Spatial Data on the Web Interest Group is a W3C entity matched by a sub-committee of OGC's Technical Committee. Collectively, the two comprise the Joint W3C/OGC Organizing Committee, the JWOC.]]

**Comment by michael-n-cooper** [comment]
@tguild thanks for the explanation. APA is satisfied and has no further comments. Over to @brewerj to complete accessibility horizontal review.

**Comment by himorin** [comment]
no comment/request from i18n group.

**Comment by tguild** [comment]
@brewerj think we just need you to complete a11y review

**Comment by brewerj** [comment]
@tguild and @michael-n-cooper just to encourage the support for indication of accessible routing, as well as accessiblity of specific locations. Ted it looked like your comment to Michael was "could do" but is this well enough covered in your scope of work so that it could be turned into a "will do"? Hard to think of another charter where this work could be picked up, and this would have a significant impact if you're also to include that work. Can you explain more how you're envisioning this fit? Thx...

**Comment by tguild** [comment]
@brewerj can do as in done with respect to @michael-n-cooper suggestion to switch WAI for APA coordination. The routing ontology work is primarily at OGC at present and may become joint work later in this SDW group. I can see us leveraging SDW as a conduit between APA and OGC in the meantime. Some A11y of locations work is being done in Linked Data for Accessibility CG which SDW is monitoring.

**Comment by brewerj** [comment]
@tguild your suggestion makes sense and your intention is clear, thanks. I'm mainly thinking how to capture the intention. Wording could be as simple as adding 1 sent following current APA coordination description to state: 
> SDW will with facilitate APA, OGC and Linked Data for Accessibility CG on accessibility considerations of SDW locations work. 
What think ye, and @michael-n-cooper does that work for you on behalf of APA?

**Comment by tguild** [comment]
@brewerj I'm not sure we need to be that detailed in capturing intention, especially since we may deviate from it. Higher level APA liaison makes sense imho without getting into that level of specifics in charter. That may delay charter, getting review/approval from Chairs, group and OGC.

**Comment by brewerj** [comment]
Hi Ted, I believe that I had flipped the "needs resolution" label to "completed" label for accessibility at the same time as I added the thumbs up to your March 3rd comment. However, I think that until @michael-n-cooper confirms this in APA's GH repo for horizontal issues, @plehegar has things set up so that it will auto-revert to the "needs resolution" label. So, Michael, can you please confirm in your APA horizontal review repo that the status on this item should be "completed" for accessibility? Thanks.
